docs/generators/typescript-angular.md

```diff
@@ -94,45 +94,35 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 ## RESERVED WORDS
 
 <ul class="column-ul">
-<li>abstract</li>
```

Comment:
lets keep the previous ones in order to maintain backwards compatibility

Reply:
yes, we should keep `byte` etc even though they wouldn't actually need to be reserved, but just to make sure that the generated code doesnt change for existing users.

Reply:
I see. The intent for this PR is to un-break prefixed reserved words, that are not really reserved words. E.g. currently I have a property that is generated as `_from`, even though it should just be `from`. The reason I am trying to change this is because it means different generators don't align.
For example, I am using an OpenAPI client to fetch data from X, it gets returned with a field `_from`. I then try to serialize it into an Avro container. The avro schema is generated with this project as well, however the schema has `from` unprefixed. Because of this discrepancy, the JSON object can't be serialized into the avro container, because `_from` != `from`. For the actual reserved properties this might be unavoidable, however for the ones that are not actually reserved it just manifests as a bug/inconvenience. Thus this pull request. I agree it breaks backwards compatibility, so maybe it should be filed against 8.x, however first we have to agree that this should be changed. If we want to keep the current behaviour and "reserved words", then I will just close this pull request.

Reply:
yeah I guess the big problem is that property names that are reserved keywords are changed for models, making them incompatible with the actual reality. so for those situations probably developers need to do someting like `(response as any).from`, so maybe it wouldn't even be a breaking change.
for parameter names that are merged into a single object like `myApi({_from: '2024-01-01'})`, this would become `myApi({from: '2024-01-01'})` and thus not be backwards compatible. lets target v8.0 for this PR

Reply:
OK. Do you have any guidance on:

> One thing I am not 100% sure us the [contextual keywords](https://github.com/microsoft/TypeScript/blob/main/src/compiler/types.ts#L186-L224), however the playground suggests they are fine to be used as identifiers.

?

Reply:
ok if its fine in the playground lets not include those